ReactWindows/Playground.Net46/Playground.Net46.csproj

```diff
@@ -160,11 +157,23 @@
       <Project>{22cbff9c-fe36-43e8-a246-266c7635e662}</Project>
       <Name>ReactNative.Net46</Name>
     </ProjectReference>
+    <ProjectReference Include="..\ReactNative.Net46\ReactNative.Net46.csproj">
+      <Project>{22cbff9c-fe36-43e8-a246-266c7635e662}</Project>
+      <Name>ReactNative.Net46</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)' == 'ReleaseBundle' or '$(Configuration)' == 'DebugBundle'">
+    <Content Include="ReactAssets\**\*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="AfterBuild">
-    <ItemGroup>
-      <ChakraCoreDll Include="$(SolutionDir)..\ChakraCore\Build\VcBuild\bin\$(Platform)_$(Configuration)\ChakraCore.*" />
+    <ItemGroup Condition="'$(Configuration)' == 'ReleaseBundle' or '$(Configuration)' == 'Release'">
```

Comment:
This AfterBuild step was not working for bundle builds because we were using the $(Configuraiton) variable as part of the path creation... This caused us to look for a folder  ..\x64_ReleaseBundle in the chakra build, which doesn't exist... The change is to put a conditional on the ChakraCoreDLL property that will append "Release" or "Debug" in both the Bundle and non bundle configuraitons.